getMoleInfo_Ensembl.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import re
+import time
+import requests
+from Bio.Seq import Seq
+from Bio import SeqIO
+import json  # Add this import at the top of your script
+
+def getMoleculeInfo(ensembl_gene_id: str) -> dict:
+    """
+    Given an Ensembl gene ID, fetch and return information about the molecule.
+
+    Params
+    ------
+    ensembl_gene_id
+        The Ensembl gene ID to lookup.
+
+    Returns
+    -------
+    A dictionary containing information about the molecule.
+    """
+
+    ensembl_server = 'http://rest.ensembl.org'
+    endpoint = f'/lookup/id/{ensembl_gene_id}?expand=1'
+    headers = {'Content-Type': 'application/json'}
+    
+    print("Sending request to:", ensembl_server + endpoint)
+    response = requests.get(ensembl_server + endpoint, headers=headers)
+    
+    print("Response status code:", response.status_code)  # Debugging
```

Comment:
going to purposefully be annoying and say use logger instead of print. Additionally think about if logger.info or logger.debug would be better :)

Reply:
Sure thing! I'm guessing logger.info would be better for the top print statement since it's an indicator to the user that the program is reaching out to the server. And the bottom statement would be logger.debug because that information is really only for me?